spec/test_app/app/models/work.rb

```diff
@@ -3,5 +3,6 @@
 # Generated by hyrax:models:install
 class Work < ActiveFedora::Base
   include ::Hyrax::WorkBehavior
-  property :creator, predicate: ::RDF::Vocab::DC.creator
+  property :single_object, predicate: ::RDF::Vocab::DC.creator, multiple: false
+  property :multiple_objects, predicate: ::RDF::Vocab::DC.creator
```

Comment:
Is `::RDF::Vocab::DC.creator` still correct with this change? I'm not sure how much it matters since this is the test work